misk/src/main/kotlin/misk/eventrouter/LocalClusterConnector.kt

```diff
@@ -0,0 +1,19 @@
+package misk.eventrouter
+
+import misk.web.actions.WebSocket
+import misk.web.actions.WebSocketListener
+import javax.inject.Singleton
+
+@Singleton
+internal class LocalClusterConnector : ClusterConnector {
+  override fun joinCluster(topicPeer: TopicPeer) {
+    topicPeer.clusterChanged(ClusterSnapshot(listOf("local"), "local"))
```

Comment:
Should use the kubernetes config to get `my_pod_name`

Reply:
`local` or `Localhost` ? Are these synonyms?